tests/sentry/snuba/metrics/test_query.py

```diff
@@ -219,6 +206,108 @@ def test_validate_order_by_field_in_select():
     MetricsQuery(**metrics_query_dict)
 
 
+def test_validate_many_order_by_fields_in_select_fails():
```

Comment:
```suggestion
def test_validate_many_order_by_fields_in_select_fails():
    metric_field_1 = MetricField(op=None, metric_name=SessionMetricKey.CRASH_FREE_RATE.value)
    metric_field_2 = MetricField(op="p50", metric_name=TransactionMetricKey.DURATION.value)
```

This should fail because session crash free rate is generated from counters entity while p50 of duration will go to distribution

Reply:
@ahmedetefy in `metric_field_2 = MetricField(op="sum", metric_name=SessionMetricKey.SESSION.value)` i see `AttributeError: SESSION`,  should i switch to any from this list?

https://github.com/getsentry/sentry/blob/f56ac68074d99791994f22c7fc85e65467645ba6/src/sentry/snuba/metrics/naming_layer/public.py#L51-L37

Reply:
resolved in a4dccd92de